user/pages/04.contributing/docs.md

```diff
@@ -31,6 +31,15 @@ You can also [create an
 account](https://mycroft-ai-slack-invite.herokuapp.com) for our Slack 
 channel and chat with other Mycroft developers.
 
+Channels:
+
+ - **#troubleshooting** (errors and general code issues)
+ - **#futures** (future code chat and dev code sharing)
+ - **#development** (github log)
```

Comment:
`#development` can also be used to discuss general dev stuff, although it isn't as frequently used.